### DISCUSSION TITLE
I want a feature for handling PipeCD configuration files

### DISCUSSION THREAD

**Discussion by ouchi2501** [pipe-cd/pipecd#4764] [Ideas]
I'm currently considering using PipeCD for Lambda's Continuous Delivery, and I'd like to customize the app.pipecd.yml and function.yml files with environment-specific values (e.g., prd, dev).
 I think it would be beneficial to have configuration management features similar to Helm, where templates are provided, and values are managed using files like values.yaml to fill in the necessary values.

**Comment by t-kikuc** [comment]
Maybe this would be useful as a reference:)

Generating config files by CUE+Go
https://speakerdeck.com/kuro_kurorrr/cue-plus-godean-quan-katujian-dan-nishe-ding-huairuwozi-dong-sheng-cheng-sitemita

**Comment by t-kikuc** [comment]
I also think this is the important enhancement.
In cdCon-2023 in Japan, 
there was a session on struggling with a lot of manifests of ArgoCD by CUE language.

> "Pipeline as Code" Journey - Jun Makishi & Kasuki Aizawa, NTT Communications
- EventPage: https://events.linuxfoundation.org/open-source-summit-japan/program/schedule/
- PDF: https://static.sched.com/hosted_files/ossjapan2023/ba/20231204_Pipeline-as-Code_%20Journey_JunMakishi_KazukiAizawa.pdf

**Comment by ffjlabo** [comment]
@nnnkkk7 We are interested in your idea to use CUE, so plz tell us more 🙏
